filer/settings.py

```diff
@@ -253,3 +253,6 @@ def update_server_settings(settings, defaults, s, t):
 FILER_DUMP_PAYLOAD = getattr(settings, 'FILER_DUMP_PAYLOAD', False)  # Whether the filer shall dump the files payload
 
 FILER_CANONICAL_URL = getattr(settings, 'FILER_CANONICAL_URL', 'canonical/')
+
+FILER_FILE_EXTENSION_ACCEPTED = ['.jpg', '.jpeg', '.png', '.gif', '.svg']
+FILER_FILE_EXTENSION_NOTHUMBS = getattr(settings, 'FILER_FILE_EXTENSION_NOTHUMBS', ['.svg'])
```

Comment:
Just a proposal:
Why not using the MIME-type instead of the file extension to distinguish thumbnailable files.
In my optionion, the MIME-type is the canonical and unique representation of a file type and it can be easily determined in Python.

Reply:
We could use [puremagic](https://pypi.org/project/puremagic/) (Pure python implementation of identifying files based off their magic numbers)

Reply:
Another approach would be to extend the `filer.models.filemodels.File` and add an extra field `mime_type`. This field then would have to be filled after uploading a file using the HTTP request header `Content-Type: ...`.
This in my opinion would be the cleanest solution, but requires 
* an approved API change request from a core maintainer of django-filer.
* a migration path, to fill this new field for all existing installations. Here we could use [puremagic](https://pypi.org/project/puremagic/).

@FinalAngel if you approve this proposal, I could implement this feature.